src/app_service/common/types.nim

```diff
@@ -14,7 +14,7 @@ type
     Audio = 8
     Community = 9
     Gap = 10
-    Edit = 11
+    ContactRequest = 11
```

Comment:
So, what's the int for Edit then? It should still be supported

Reply:
I don't see any types named as `Edit` in status-go. In our code it was not used.